config/sync/core.entity_view_display.comment.os2loop_post_comment.default.yml

```diff
@@ -12,6 +12,16 @@ targetEntityType: comment
 bundle: os2loop_post_comment
 mode: default
 content:
+  flag_os2loop_upvote_correct_answer:
+    weight: 10
+    settings: {  }
+    third_party_settings: {  }
+    region: content
+  flag_os2loop_upvote_upvote_button:
+    weight: 10
+    settings: {  }
+    third_party_settings: {  }
+    region: content
```

Comment:
These fields seem to duplicate the`os2loop_upvote_flag_…`-fields. Should they be removed?